src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisoners/PrisonersResourceIntTest.kt

```diff
@@ -31,6 +32,7 @@ class PrisonersResourceIntTest : IntegrationTestBase() {
   }
 
   @Test
+  @Transactional
```

Comment:
@andymarke found out that I could just add transactional to the tests that were moaning about no session